bus_presence_override/static/src/less/systray.less

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+#userStatus {
+    &:hover i {
+        color: #d3d3d3 !important;
+    }
+
+    i {
+        padding-top: 2px;
+    }
+}
+
+.o_user_presence_dropdown {
```

Comment:
CSS classes should be hyphen separated instead of snake/camel case. `.o-user-presence-dropdown`